eng/Versions.props

```diff
@@ -25,8 +25,8 @@
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
     <DefaultNetFxTargetFramework>net472</DefaultNetFxTargetFramework>
-    <DefaultNetCoreTargetFramework>net7.0</DefaultNetCoreTargetFramework>
-    <DefaultNetCoreWindowsTargetFramework>net7.0-windows</DefaultNetCoreWindowsTargetFramework>
+    <DefaultNetCoreTargetFramework>net7.0;net8.0</DefaultNetCoreTargetFramework>
```

Comment:
Do we want net8 to be first, so we get the most up to date null annotations by default in the IDE?

At least, I think thats how that works